fhir-server-test/src/test/java/com/ibm/fhir/server/test/PlanDefinitionApplyOperationTest.java

```diff
@@ -330,10 +336,8 @@ public Parameters generateParameters(String planDefinition, List<String> subject
      * @param settingContext
      * @return
      */
-    public List<ApplyOperationResult> runValidTest(String planDefinition, List<String> subject,
-        String encounter, String practitioner,
-        String organization, String userType, String userLanguage, String userTaskContext,
-        String setting, String settingContext) {
+    public List<ApplyOperationResult> runValidTest(String planDefinition, List<String> subject, String encounter, String practitioner, String organization,
+        String userType, String userLanguage, String userTaskContext, String setting, String settingContext) {
```

Comment:
same as earlier comment
```suggestion
    public List<ApplyOperationResult> runValidTest(String planDefinition, List<String> subject, String encounter, String practitioner, String organization,
            String userType, String userLanguage, String userTaskContext, String setting, String settingContext) {
```